### DISCUSSION TITLE
E1 Zoom?

### DISCUSSION THREAD

**Discussion by nurimoca** [fwestenberg/reolink_dev#476] [Q&A]
Does anyone know if it works with the E1 ZOOM?

**Comment by martinberteig** [comment]
I have two E1 Zoom cameras. They are working without any issues.  :)
Note that my usage is very limited. I have just added the cameras through the integration and showing the feed on the dasboard. Not using any of the other entities, so i havent tested them.

**Reply by conma293** [reply]
is there a siren entity available or you have to go through the reolink app?

**Reply by tcholston** [reply]
I had two E1 Zooms on that integration. Then they failed. Now I cannot get them back. Had to switch to ONVIF to view them, but miss all the sensors.

**Reply by PowoPower** [reply]
Hi, somebody can confirm that the E1 ZOOM is ONVIF pls?
